inlineplz/linters/__init__.py

```diff
@@ -55,27 +59,703 @@ def vendored_path(path):
     )
 
 
-# glob patterns for what language is represented by what type of files.
-PATTERNS = {
-    "all": ["*.*"],
-    "ansible": ["*.yaml", "*.yml"],
-    "docker": ["*Dockerfile", "*.dockerfile"],
-    "gherkin": ["*.feature"],
-    "go": ["*.go"],
-    "groovy": ["*.groovy", "Jenkinsfile", "jenkinsfile"],
-    "java": ["*.java"],
-    "javascript": ["*.js"],
-    "json": ["*.json"],
-    "html": ["*.html", "*.htm"],
-    "markdown": ["*.md"],
-    "python": ["*.py"],
-    "shell": ["*.sh", "*.zsh", "*.ksh", "*.bsh", "*.csh", "*.bash"],
-    "stylus": ["*.styl"],
-    "robotframework": ["*.robot"],
-    "rst": ["*.rst"],
-    "text": ["*.md", "*.txt", "*.rtf", "*.html", "*.tex", "*.markdown"],
-    "yaml": ["*.yaml", "*.yml"],
-}
+GROOVY_PATH = vendored_path(os.path.join("groovy", "groovy-all-2.4.15.jar"))
+SLF4J_PATH = vendored_path(os.path.join("groovy", "slf4j-api-1.7.25.jar"))
+
+
+def register_patterns():
```

Comment:
I actually kinda like these as hardcoded constants as long as they aren't dynamic, though they could probably be pulled out into a separate file.